singularity-opac/photons/epbremsstrahlung_opacity_photons.hpp

```diff
@@ -177,8 +177,7 @@ class EPBremsstrahlungOpacity {
   Real mass_ion_;
   PlanckDistribution<pc> dist_;
   static constexpr Real mmw_ =
-      (pc::mp + pc::me) /
-      (2. * pc::mp); // Neutral fully ionized electron-proton gas
+      (pc::mp + pc::me) / 2.; // Neutral fully ionized electron-proton gas (g)
```

Comment:
double checking---is this change right? Going from unitless to grams?

Reply:
yes, this makes more sense because now `ne = ni  = rho / mmw`